admin-ui/plugins/auth-server/components/Scopes/ScopeForm.js

```diff
@@ -31,38 +31,67 @@ import GluuTooltip from 'Routes/Apps/Gluu/GluuTooltip'
 import { SCOPE } from 'Utils/ApiResources'
 import { useTranslation } from 'react-i18next'
 import { ThemeContext } from 'Context/theme/themeContext'
-import {
-  LIMIT,
-  PATTERN,
-} from 'Plugins/auth-server/common/Constants'
+import { LIMIT, PATTERN } from 'Plugins/auth-server/common/Constants'
+import { getUMAResources } from '../../redux/actions/ScopeActions'
 
-function ScopeForm({ scope, scripts, attributes, handleSubmit}) {
+function ScopeForm({ scope, scripts, attributes, handleSubmit }) {
   const { t } = useTranslation()
   let dynamicScopeScripts = []
+  let umaResourcesScript = []
+  let umaAuthorizationPolicies = []
+
   const theme = useContext(ThemeContext)
   const selectedTheme = theme.state.theme
   const history = useHistory()
   const spontaneousClientScopes = scope.attributes.spontaneousClientScopes || []
-   const dispatch = useDispatch()
-   const client = useSelector((state) => state.oidcReducer.items)
+  const dispatch = useDispatch()
+  const client = useSelector((state) => state.oidcReducer.items)
```

Comment:
This is not required. Please populated associated clients for UMA and spontaneous scopes from scope obj.

Reply:
done